Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Paged/ContentBlockerScriptHandler.swift

```diff
@@ -90,7 +90,8 @@ extension ContentBlockerHelper: TabContentScript {
           sourceURL: sourceURL,
           enabledRuleTypes: genericTypes,
           resourceType: dto.data.resourceType,
-          isAggressiveMode: domain.blockAdsAndTrackingLevel.isAggressive
+          isAggressiveMode: domain.blockAdsAndTrackingLevel.isAggressive,
+          domain: domain
         )
```

Comment:
I noticed the domain wasn't being checked. It's not a big deal because this responds to scripts that wouldn't be injected anyway, but figured should fix it.